clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/v1/provider/view/KubernetesLoadBalancerProvider.groovy

```diff
@@ -121,16 +121,26 @@ class KubernetesLoadBalancerProvider implements LoadBalancerProvider<KubernetesL
     }
   }
 
-  // TODO(lwander): Implement if/when these methods are needed in Deck.
   @Override
   LoadBalancerProvider.Item get(String name) {
-    throw new OperationNotSupportedException("Kubernetes is a special snowflake.")
+    // this is just a placeholder
+    return list().find { it.name == name }
```

Comment:
Yeah I'm not sure what this method is supposed to do, any ideas @ttomsu?

Reply:
It's a getter based on LB name... where's the confusion?

IIRC, @anotherchrisberry once said that LBs have to have unique names. I think this endpoint might be used to that end.

Reply:
Typically you need an account & region to make it unique.

Reply:
Yep. We have Load Balancers with the same name across multiple clusters so something like `myapp-internal` exists multiple times in the list.

Reply:
@lwander @ttomsu I _do_ know that `objectMapper.converValue` uses it. I guess it's supposed to get more details attributes?

Reply:
Paging the @ttomsu - is it really the case that AWS LBs are unique across all accounts and regions? (even ones you don't own)

Reply:
Or @anotherchrisberry

Reply:
AWS ELB names do not have to be unique across account/regions, but they do need to be unique across all the _VPCs_ in a given account/region, which might be where the confusion is coming from here.